.github/workflows/build_and_deploy.yml

```diff
@@ -45,13 +45,19 @@ jobs:
           elif [ '${{ github.event_name }}' == 'workflow_dispatch' ]
           then
             echo "value=force-preview" >> $GITHUB_OUTPUT
+          elif [[ $(node scripts/run-for-change.js --not --type docs --exec echo 'false') != 'false' ]];
+          then
+            echo "value=skipped" >> $GITHUB_OUTPUT
           else
             echo "value=automated-preview" >> $GITHUB_OUTPUT
           fi
       - name: Print deploy target
         run: echo "Deploy target is '${{ steps.deploy-target.outputs.value }}'"
 
   build:
+    if: ${{ needs.deploy-target.outputs.value != 'skipped' }}
+    needs:
+      - deploy-target
```

Comment:
Increases wall-time of publishing the tarballs. <15s should be fine. Deploy tests might fail though since they rely on current timing

Reply:
Can we ensure we don't break deploy-tests as those could be quite noisy when failing

Reply:
Not really because they rely implicitly on timing. If 10s increased wall time is an issue, we're already close to breakage. The spread of the wall time ranges from 2-6mins so an added 10s should not make a difference.

Reply:
Ah yea 10s should be fine hopefully